docs/articles/clustering/member-roles.md

```diff
@@ -0,0 +1,47 @@
+---
+uid: member-roles
+title: Member Roles
+---
+
+# Why Are Roles Important
+
+A cluster can have multiple Akka.NET applications in it, "roles" help to distinguish different Akka.NET applications within a cluster!
+
+# How Can Roles Help
+
+Not all Akka.NET applications in a cluster need to perform the same function. For example, there might be one sub-set which runs the web front-end, one which runs the data access layer and one for the number-crunching. 
+Choosing which actors to start on each node, for example cluster-aware routers, can take member roles into account to achieve this distribution of responsibilities.
+
+# Usage
```

Comment:
Move this to right after your opening sentence in the document, drop the `#Usage`